controllers/packagebundlecontroller_controller_test.go

```diff
@@ -67,162 +66,130 @@ func TestPackageBundleControllerReconcilerReconcile(t *testing.T) {
 		}
 	}
 
-	t.Run("marks status inactive if name doesn't match", func(t *testing.T) {
+	t.Run("happy path", func(t *testing.T) {
 		t.Parallel()
 
 		ctx := context.Background()
 		mockClient := mocks.NewMockClient(gomock.NewController(t))
 		pbc := givenPackageBundleController()
-		pbc.Namespace = "blah"
-		inactiveController := types.NamespacedName{
-			Name:      "blah",
-			Namespace: api.PackageNamespace,
-		}
-		mockClient.EXPECT().Get(ctx, inactiveController, gomock.Any()).
+
+		mockClient.EXPECT().Get(ctx, req.NamespacedName, gomock.Any()).
 			DoAndReturn(setMockPBC(&pbc))
-		mockStatusClient := mocks.NewMockStatusWriter(gomock.NewController(t))
-		mockClient.EXPECT().Status().Return(mockStatusClient)
-		mockStatusClient.EXPECT().Update(ctx, gomock.Any(), gomock.Any()).
-			DoAndReturn(func(ctx context.Context, pbc *api.PackageBundleController,
-				opts *client.UpdateOptions) error {
-				if pbc.Status.State != api.BundleControllerStateIgnored {
-					t.Errorf("expected state to be set to Ignored, got %q",
-						pbc.Status.State)
-				}
-				return nil
-			})
+		mockBundleManager := bundleMocks.NewMockManager(gomock.NewController(t))
+		mockBundleManager.EXPECT().ProcessBundleController(ctx, &pbc).Return(nil)
 
-		r := NewPackageBundleControllerReconciler(mockClient, nil, bm,
+		r := NewPackageBundleControllerReconciler(mockClient, nil, mockBundleManager,
 			logr.Discard())
-		result, err := r.Reconcile(ctx, ctrl.Request{NamespacedName: inactiveController})
+		result, err := r.Reconcile(ctx, req)
 		if err != nil {
-			t.Fatalf("expected no error, got %s", err)
-		}
-		if result.Requeue {
-			t.Errorf("expected Requeue to be false, got true")
+			t.Errorf("expected no error, got %s", err)
 		}
+		assert.True(t, result.Requeue)
 	})
 
-	t.Run("leaves state as is", func(t *testing.T) {
+	t.Run("process error", func(t *testing.T) {
 		t.Parallel()
 
 		ctx := context.Background()
 		mockClient := mocks.NewMockClient(gomock.NewController(t))
 		pbc := givenPackageBundleController()
-		pbc.Status.State = api.BundleControllerStateActive
 
 		mockClient.EXPECT().Get(ctx, req.NamespacedName, gomock.Any()).
 			DoAndReturn(setMockPBC(&pbc))
-		testBundle := api.PackageBundle{}
 		mockBundleManager := bundleMocks.NewMockManager(gomock.NewController(t))
-		mockBundleManager.EXPECT().LatestBundle(ctx, pbc.Spec.Source.BaseRef()).Return(&testBundle, nil)
-		mockBundleManager.EXPECT().ProcessLatestBundle(ctx, &testBundle).Return(nil)
+		mockBundleManager.EXPECT().ProcessBundleController(ctx, &pbc).Return(fmt.Errorf("oops"))
 
 		r := NewPackageBundleControllerReconciler(mockClient, nil, mockBundleManager,
 			logr.Discard())
-		_, err := r.Reconcile(ctx, req)
+		result, err := r.Reconcile(ctx, req)
 		if err != nil {
 			t.Errorf("expected no error, got %s", err)
 		}
+		assert.True(t, result.Requeue)
 	})
 
-	t.Run("marks status disconnected if active", func(t *testing.T) {
+	t.Run("marks status ignored bogus name", func(t *testing.T) {
```

Comment:
This doesn't mention what was being tested and a little more description would be great.